### DISCUSSION TITLE
你們的morning routine是什麼

### DISCUSSION THREAD

**Discussion by roma0323** [1101-GS4719-C/Python-Community-GS4719-C#1540] [Q&A]
許多人說早上起床的那一個小時是最精華的，應該排自己的morning routine。
我自己目前是喝水、刷牙、偶爾伏地挺身、想今天一整天的規劃。
曾經嘗試過要學網路上的那些勵志影片說要冥想讀書什麼的，但真的做不太到。
好奇其他人的mornings routine 都長什麼樣子呢？

**Comment by Boying12** [comment]
Creating a morning routine can set a positive tone for your day and help you feel more organized and productive. Here’s my morning routine. 
1. Hydrate (5 minutes)
- Drink a glass of water 
2. Mindfulness or Meditation (10 minutes)
-actually take a nap (XD
4. Personal Hygiene (20 minutes)
- Take a shower, brush your teeth, and get dressed for the day
5. Plan Your Day (10 minutes)
6. Read or Learn Something New (10-15 minutes)

**Comment by Denny3908** [comment]
1. 冥想（一定要在看手機之前，不然就會一直滑下去XD） 5mins
2. 刷牙、洗臉 ~3mins
3. 拉筋和一點點的運動（不會流汗的那種，like壓腿、倒立）~15mins
4. Coffeeeeeee 有時候泡即溶有時候買便利商店 ~5mins
5. 回mail、訊息、排行事曆和代辦 ~30mins

**Comment by DarsooK** [comment]
1. 下床 3min
2. 穿內衣內褲 30sec
3. 打開網路機上盒 1sec
4. 喝水、上廁所 1min
5. 滑手機 10min
6. 著衣 5min
7. 通勤 30min

**Comment by ed112601513** [comment]
1.賴床(0~10min)
2.上廁所、刷牙、洗臉，再換衣服(10min)
3.滑手機，確認今天要做什麼和該帶什麼(3min)
4.整理背包(3min)
5.買早餐後去上課(20min)

**Comment by Jing-Asteroid** [comment]
睜眼
把棉被翻開
轉身
上半肢驅動銜接下肢起身，也就是俗稱的起床
邁動雙腿
朝向廁所
執行刷牙的動作
結束刷牙的動作
邁動雙腿
朝向床鋪
下肢驅動銜接上半肢躺下去，也就是俗稱的回籠覺
轉身躺好
把棉被蓋好，別著涼了
閉眼
繼續睡

**Reply by Denny3908** [reply]
Dream life bruh

**Comment by Tchang323** [comment]
1.刷牙、洗臉
2. 閱讀20min(有時間的話)
3. 吃早餐
4. 化妝
5. 換衣服出門

**Comment by lai1008** [comment]
1.吃藥
2.上廁所、刷牙、洗臉，再換衣服(10min)
3.滑手機，確認今天要做什麼和該帶什麼(3min)
4.整理背包(3min)
5.買早餐後去上課(20min)

**Comment by 112121py** [comment]
1. 賴床掙扎
2. 拿起手機滑一下，清醒腦袋
3. 刷牙上廁所
4. 穿個衣服就可以出門了
3-4那邊會邊想自己今天會做什麼事要上什麼課要拿什麼東西，唸一下手機錢包鑰匙看有沒有帶，就出門了xd

**Comment by wcyeee** [comment]
起床先馬上換衣服，然後去刷牙洗臉，之後去煮早餐！
我覺得煮早餐是一個很幸福的醒腦行程！
過程需要注意力集中，結束之後還有好吃的食物可以吃。
之後吃外早餐就差不多可以出門或開始休息了！

**Comment by Teddy111802518** [comment]
基本上沒有什麼流程
但大致上
起床
上廁所
打開電腦
開始專題/去上課

沒什麼 喔還去運動 或冥想

**Comment by Jinyuu17** [comment]
1. 起床和伸展：許多人喜歡早上伸展身體或進行簡單的瑜伽來喚醒肌肉和關節。
2. 喝水或咖啡：補充水分或喝一杯咖啡來提神。
3. 運動：一些人會進行晨跑、健身或其他形式的運動，以促進血液循環和增加能量。
4. 早餐：一份健康的早餐，富含蛋白質和纖維，有助於為接下來的工作提供能量。
5. 規劃一天：有些人會花幾分鐘查看行程表，設定當天的目標，或進行短時間的冥想來整理思緒。

這樣的例行活動能幫助我開始一天時更有條理和精力充沛。

**Comment by qiu0109** [comment]
1.坐著滑手機直到清醒或時間不夠
2.上廁所換衣服
3.出門
雖然我知道不吃早餐很不健康，但是到了大學就會只想睡多一點，拖到最後一刻才要出門上課根本沒時間吃早餐。

**Comment by 113607514** [comment]
我也聽過很多人說早上那個小時很重要，得好好規劃，來決定一天的基調。我現在的早晨常規基本是喝水、刷牙，有時候會做幾個伏地挺身，然後就開始想今天該做什麼，看看有沒有什麼需要優先處理的事情。雖然也試過看一些網上的勵志影片，想學冥想或讀書，但說實話我做不太到，還是覺得把重點放在讓自己舒服並有效率的開始一天更實際。你呢，會有什麼特別的早晨習慣嗎？

**Comment by Taoyi-Chiang** [comment]
喝咖啡等回神。沒有喝到咖啡就沒辦法開機。
最近實在是太冷了，morning routine幾乎就是醒來，滑手機，睡回去，反覆幾次到中午才起床。

**Comment by poyen0806** [comment]
morning routine如下

睜眼
質疑自己
睡回去
發現睡過頭
猛然睜眼
發現是自己記錯時間
繼續睡回去

重複上面流程直到真的該起床了
就醒來然後下床吃午餐

對，大學後就沒怎麼吃早餐了


如果我努力一點的話
我或許可以寫個python程式設定一個屬於自己的鬧鐘
讓我比較容易起床
但我覺得只要是鬧鐘
就很難喜歡它

**Comment by lichilun** [comment]
我的morning routine如下：

1.每天早上醒來發現天有點亮，糟了發現自己睡過頭了！
2.趕快從床上起來，手忙腳亂摸手機看時間，結果發現遲到了！
3.想說要趕快去學校，快速洗臉刷牙，但是還在半夢半醒！
4.刷到一半，想說算了已經遲到，躺回去床上...

也許未來能自己發明，一種程式，讓人們能克服生理睡眠，在定時起床
